python/composio/tools/local/codeanalysis/actions/create_codemap.py

```diff
@@ -0,0 +1,305 @@
+import json
+import os
+import shutil
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict
+
+from pydantic import BaseModel, Field
+from tqdm.auto import tqdm
```

Comment:
remove this as a dependency or make this a action dependency eg `requires=["tqdm"]`